kevinpapst/adminlte-bundle/0.9/config/packages/admin_lte.yaml

```diff
@@ -0,0 +1,45 @@
+#
+# Configuration file for the AdminLTE bundle
+#
+# For more information about the bundle settings visit:
+# https://github.com/kevinpapst/AdminLTEBundle/blob/master/Resources/docs/configurations.md
+#
+admin_lte:
+    options:
+        # default image for missing user avatar images
+        default_avatar: bundles/adminlte/images/default_avatar.png
+        # the color skin
+        skin: skin-blue
+        # if activated, the boxed_layout will be deactivated
+        fixed_layout: false
+        # boxed layout (true) or full-screen (false)
+        boxed_layout: false
+        # initial collapsed state of the sidebar
+        collapsed_sidebar: false
+        # whether the logo collapses or not
+        mini_sidebar: false
+        # configurable tabs in the control_sidebar (right screen), see:
+        # https://github.com/kevinpapst/AdminLTEBundle/blob/master/Resources/docs/control_sidebar.md
+        #control_sidebar:
+        #    home:
```

Comment:
Is this something 80% of the user will uncomment and use? 

Im not a big fan of config in comments like this.

Reply:
I have seen some of your your comments about "comments in configs" in other PRs as well. 

Two questions:
- If I remove the commented configs but leave the docu link, would that be okay?
- What about using `copy-from-package` instead of `copy-from-recipe`? I have seen that only a couple of recipes use it, but that would remove the "documentation" from the recipe repo. Would that be better?